client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -163,53 +173,55 @@ class Dashboard extends Component {
 		return (
 			<div>
 				{ manageView }
-				{ ! this.props.mailChimpConfigured &&
-					( config.isEnabled( 'woocommerce/extension-settings-email' ) && (
-						<MailChimp site={ selectedSite } redirectToSettings dashboardView />
-					) ) }
+				{ ! this.props.mailChimpConfigured && (
+					<MailChimp site={ selectedSite } redirectToSettings dashboardView />
+				) }
```

Comment:
This is an unrelated change where a `config.isEnabled` check is being removed. If you and @budzanowski can confirm that this change is wanted, this PR is ready to merge.

Reply:
I confirmed with @budzanowski via DM that he is okay with this change - thanks for asking about it though!